src/statistics/Fis.ts

```diff
@@ -63,8 +63,8 @@ export class Fis implements StatisticsProvider {
       } else {
         payload.evalscript = Buffer.from(layer.getEvalscript(), 'utf8').toString('base64');
       }
-      if (layer instanceof AbstractSentinelHubV1OrV2Layer) {
-        payload.evalsource = layer.getEvalsource();
+      if (layer.type === 'AbstractSentinelHubV1OrV2Layer') {
```

Comment:
Yes it can be, with other compromise needed and some more work. And AbstractSentinelHubV1OrV2Layer does not support Statistical API due to change on BE  https://docs.sentinel-hub.com/api/latest/api/statistical/#statistical-api-and-evalscript , also StatisticalApi interface was indicating that.

Reply:
please see cd commit reworked https://github.com/sentinel-hub/sentinelhub-js/pull/268/commits/1d4ed815828aa65819fa096074edebfa67da98c8

Reply:
Please see https://github.com/sentinel-hub/sentinelhub-js/pull/268/commits/dbd239ab0e997f83ab29749bd1d9fdb8a132cab4 for rework